### ISSUE TITLE
Post Carousel block: Enlarged/spilled over the page for Atomic site

### ISSUE THREAD

**Issue by retnonindya** [Automattic/wp-calypso#54340]
Hi team! I feel this is better to be filed under https://github.com/Automattic/newspack-blocks but Bugomattic brings me here, so... 😅 

So far, we have received three reports -- all Atomic sites -- of Carousel block enlarging and spilled over the page.

This is one example:

![Screenshot 2021-07-07 at 8 51 00 AM](https://user-images.githubusercontent.com/44726232/124683796-76240c00-df00-11eb-9ad8-20e6d926c21a.png)

We are still not sure what caused this, but I found temporary CSS workaround to ensure the Post Carousel block "fixed":

```
.wp-block-newspack-blocks-carousel {
    max-width: 100%;
}
.swiper-container.swiper-container-initialized.swiper-container-horizontal {
    max-width: 100%;
}
```

Thank youuu.

#### Does this happen on simple or atomic sites or both?
Atomic. All themes.

#### Is there any console output or error text?
No.

#### Level of impact (Does it block purchases? Does it affect more than just one site?)
Multiple sites

#### Slack report
p1625618115176600-slack-C03TY6J1A

**Comment by JoshuaGoode** [comment]
I'm not in to test but I suspect this might be due to Jetpack 9.9 which released today on Atomic.

full-site-editing hasn't been updated in 7 days.

**Comment by JoshuaGoode** [comment]
On my phone but confirmed on a test site that it is Jetpack.

Seems that it overrides the block's width styling with its own `.swiper-container ` styling.

This impacts plugins and themes that use `.swiper-container`.

For example: Madd Magazine is a third-party theme that uses `.swiper-container` and breaks.

I'm guessing this is coming from Jetpack's new usage of swiper.js

**Comment by joweber123** [comment]
I can confirm that this is affecting Elementor plugin users using the image carousel widget.  The CSS workaround above worked to temporarily fix that.

30507852-hc

**Comment by JoshuaGoode** [comment]
I won't be at a computer tonight to move this issue but the TL;DR is that Jetpack 9.9's new usage of swiper.js is now visually breaking themes and plugins that also use swiper.js.  For example, Newspack's Post Carousel block, Elementor's image carousel widget.

Related changes: https://github.com/Automattic/jetpack/pull/20107

cc: @sdixon194 @jeherve

**Comment by jeherve** [comment]
Thanks for the reports! We're tracking this in this issue, in the Jetpack repo:
https://github.com/Automattic/jetpack/issues/20289

**Comment by aleone89** [comment]
Two reports today: 

hc-20000987
hc-30683438 

The CSS above solved the issue.

**Comment by aleone89** [comment]
Another report: 

hc-30627755

**What I did**
1. Deactivated JP Carousel module in blog RC.
2. Installed Jetpack Beta > searched for and applied > update / carousel css tweaks
3. Do not activate Release Candidate in Jetpack Beta results in a blank WP-Admin

**Comment by kspilarski** [comment]
Same case in 4171236-zd. Provided CSS workaround.

**Comment by sdixon194** [comment]
@kspilarski this should have been fixed with Jetpack 9.9.1 which was released on Atomic late last Tuesday. I took a look and didn't see the issue on the site anymore, and I don't think they applied the CSS yet. It also looks like they have some performance/caching plugins. I'd reckon the site's cache wasn't cleared and recommend they do that/disable the plugins to see if that fixes things on their end.
